ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,187 @@
+    if otel_value != "none":
+        log.warning(
+            "An unrecognized runtime metrics exporter '%s' is being "
+            "used; setting DD_RUNTIME_METRICS_ENABLED to False.",
+            otel_value,
+        )
+    return "False"
+
+
+def _remap_logs_exporter(otel_value):
+    """Logs warning when OTEL Logs exporter is configured. DDTRACE does not support this configuration."""
+    if otel_value != "none":
+        log.warning(
+            "Unsupported OTEL logs exporter value detected: %s. Only the 'none' value is supported.", otel_value
+        )
+        return ""
+    return ""
+
+
+def _remap_otel_tags(otel_value):
+    """Remaps the otel tags to ddtrace tags"""
+    dd_tags = []
+    remaining_tags = []
+    otel_tags = otel_value.split(",")
+    otel_user_tag_dict = dict()
+
+    try:
+        for tag in otel_tags:
+            tag_pair = tag.split("=")
+            otel_user_tag_dict[tag_pair[0]] = tag_pair[1]
+
+        for otel_key, dd_key in OTEL_UNIFIED_TAG_MAPPINGS.items():
+            if otel_key in otel_user_tag_dict.keys():
+                dd_tags.append("{}:{}".format(dd_key, otel_user_tag_dict[otel_key]))
+
+        for key, value in otel_user_tag_dict.items():
+            if key.lower() not in OTEL_UNIFIED_TAG_MAPPINGS.keys():
+                if len(dd_tags) < 10:
+                    dd_tags.append("{}:{}".format(key, value))
+                else:
+                    remaining_tags.append("{}:{}".format(key, value))
+    except Exception:
+        log.warning("DDTRACE failed to read OTEL_RESOURCE_ATTRIBUTES. This value is misformatted: %s", otel_value)
+
+    if len(otel_user_tag_dict.items()) > 10:
+        log.warning(
+            "To preserve metrics cardinality, only the following first 10 tags have been processed %s. "
+            "The following tags were not ingested: %s",
+            dd_tags,
+            remaining_tags,
+        )
+    return ",".join(dd_tags)
+
+
+def _remap_otel_sdk_config(otel_value):
+    """Remaps the otel sdk config to ddtrace sdk config"""
+    if otel_value == "false":
+        return "True"
+    elif otel_value == "true":
+        return "False"
+    else:
+        log.warning("OTEL_SDK_DISABLED='%s'  is not supported", otel_value)
+        return otel_value
+
+
+def _remap_default(otel_value):
+    """Remaps the otel default value to ddtrace default value"""
+    return otel_value
+
+
+ENV_VAR_MAPPINGS = {
+    "OTEL_SERVICE_NAME": ("DD_SERVICE", _remap_default),
+    "OTEL_LOG_LEVEL": ("DD_TRACE_DEBUG", _remap_otel_log_level),
+    "OTEL_PROPAGATORS": ("DD_TRACE_PROPAGATION_STYLE", _remap_otel_propagators),
+    "OTEL_TRACES_SAMPLER": ("DD_TRACE_SAMPLE_RATE", _remap_traces_sampler),
+    "OTEL_TRACES_EXPORTER": ("DD_TRACE_ENABLED", _remap_traces_exporter),
+    "OTEL_METRICS_EXPORTER": ("DD_RUNTIME_METRICS_ENABLED", _remap_metrics_exporter),
+    "OTEL_LOGS_EXPORTER": ("", _remap_logs_exporter),  # Does not set a DDTRACE environment variable.
```

Comment:
does this need to be here if there is no mapping?

Reply:
@brettlangdon Left in as the mapped function logs a warning if an improper value is passed-- is that alright?

Reply:
all good, the reasoning makes sense.